io/plugins/eu.esdihumboldt.hale.io.shp/src/eu/esdihumboldt/hale/io/shp/writer/ShapefileInstanceWriter.java

```diff
@@ -234,7 +235,6 @@ private Map<String, Map<String, SimpleFeatureType>> createFeatureType(
 
 			if (missingGeomsForSchemas.contains(schemaEntry.getKey())) {
 				reporter.warn("No geometry found for " + schemaEntry.getKey());
-				continue;
```

Comment:
> 1. if there are some features with geometry and some without for a feature type, only the ones with geometry end up in the target file?

ONLY the features with geometry end up in the target file.

> if there are no features with geometry for a feature type, is an empty target file created for the type, or none at all?

NO file will be created for the features without a geometry.

> or will an error be thrown?

In the last page of the wizard there is a warning `"Warning! Feature types with no geometry will not be exported"` and if there is not geometry presented at all in any of the features that at the end of the export a warning message of the reported `IOReporter `will be shown informing the user that `"No file has been exported because there was no geometry found."`

An example would be 
![image](https://github.com/halestudio/hale/assets/67077116/4c69d42c-1102-4b0b-b703-f2490a4b6479)